### PR TITLE
[IA-1653] Fix cookie setting in google_sign_in.js in terra-jupyter-base

### PR DESCRIPTION
**Pull request by rtitle**
Also update all sub-images with the new base.

See https://broadworkbench.atlassian.net/browse/IA-1653

### CODE REVIEW

**Comment by rtitle** [comment]
I still need to actually test this

File: terra-jupyter-base/custom/google_sign_in.js [hunk 1]
```diff
@@ -81,10 +81,10 @@ function startTimer() {
   setInterval(statusCheck, 60000);
 }
 
+// Note: this should match
+// https://github.com/DataBiosphere/leonardo/blob/develop/http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/CookieHelper.scala
 function setCookie(token, expiresIn) {
-  const expiresDate = new Date();
-  expiresDate.setSeconds(expiresDate.getSeconds() + expiresIn);
-  document.cookie = 'LeoToken=' + token + ';secure;expires=' + expiresDate.toUTCString() + ';path=/';
+  document.cookie = 'LeoToken=' + token + '; Max-Age=' + expiresIn + '; Path=/; Secure; SameSite=None'
```
Comment (rtitle) [inline terra-jupyter-base/custom/google_sign_in.js]: This is the actual fix

**Comment by rtitle** [comment]
So I built the image and deployed it through Terra and confirmed that token refresh works. Though it was never actually broken for me so I'm not sure how great a test that is.

@calbach would you be able to test the image `us.gcr.io/broad-dsp-gcr-public/terra-jupyter-gatk:ia-1653` through AoU and see if the issue is resolved? Once this PR is merged, the image version will become `us.gcr.io/broad-dsp-gcr-public/terra-jupyter-gatk:0.0.11`.

**Comment by calbach** [comment]
Thanks @rtitle , I'm lazy so I decided to just use chrome local overrides to paste in your updated JS on a Workbench test window rather than pulling the new image. I've verified that it is successfully updating the Max-Age parameter of the cookie which seems to indicate it is working - I will check back in >1h to verify it's not 401ing.

**Comment by rtitle** [comment]
Great, thanks!

**Comment by calbach** [comment]
Confirmed it works for at least 1.5h :+1: , thanks.

**Comment by rtitle** [comment]
Great, I also left a notebook open for hours and it seems to work. Merging.

**Pull request merged**